hw/ip/csrng/doc/_index.md

```diff
@@ -16,7 +16,7 @@ In this document the terms "DRNG" and "TRNG" are used most generally to refer to
 However, the terms "DRBG" or "NRBG" are specifically used when respectively referring to SP 800-90A or SP 800-90C requirements.
 Meanwhile, when addressing requirements which originate from AIS31 we refer to the specific DRG.3 or PTG.3 classes of RNGs. 
 
-This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [Comportable guideline for peripheral functionality]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure and software-inaccessible transmission of random numbers.
+This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [Comportable guideline for peripheral functionality]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure and software-inaccessible transmission of random numbers. The bus connections to peripheral modules is done using the CSRNG application interface. This interface allows peripherals to manage CSRNG instances, and request for obfuscated entropy to be returned from the CSRNG module.
```

Comment:
should this snippet here mention the EDN?

Reply:
A bullet item in EDN was updated to mention the connections between peripherals, EDN, and CSRNG.

Reply:
Thanks @mwbranstad. I've also added them here in response to @imphil's question about the intended users of the application interface.